Defaults.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
 	s.name = 'Defaults'
-	s.version = '2.0.2'
+	s.version = '2.1'
```

Comment:
Don’t change the version. I will do so when publishing the new version.

Reply:
```suggestion
	s.version = '2.0.2'
```